### PR TITLE
Weighted loss for Sentence Classification

### PR DESCRIPTION
**Pull request by purn3ndu**
To support imbalanced datasets in Sentence Classification model.
Usage - `-class_balancing=weighted_loss`

### CODE REVIEW

File: examples/nlp/sentence_classification_with_bert.py [hunk 1]
```diff
@@ -3,6 +3,8 @@
 
 import numpy as np
 from pytorch_transformers import BertTokenizer
+from torch import nn
```
Comment (okuchaiev) [inline examples/nlp/sentence_classification_with_bert.py]: why do we need:
```python
from torch import nn
import torch
```

Comment (okuchaiev) [reply examples/nlp/sentence_classification_with_bert.py]: please remove

Comment (purn3ndu) [reply examples/nlp/sentence_classification_with_bert.py]: need it for `class_weights = torch.FloatTensor(data_desc.class_weights).cuda()`

Comment (chiphuyen) [reply examples/nlp/sentence_classification_with_bert.py]: 1. we try not to expose pytorch code on the application script.
2. we shouldn't force anything onto cuda() because it'll break on cpu. use .to(self._device) in nemo.backends.pytorch.common.CrossEntropyLoss

Comment (purn3ndu) [reply examples/nlp/sentence_classification_with_bert.py]: Made suggested changes.

Comment (okuchaiev) [reply examples/nlp/sentence_classification_with_bert.py]: thx
